crates/graphql/src/schema.rs

```diff
@@ -555,6 +556,144 @@ impl<'a> From<models::Storefront<'a>> for Storefront {
     }
 }
 
+#[derive(Debug, Clone)]
+struct Marketplace {
+    subdomain: String,
+    name: String,
+    description: String,
+    logo_url: String,
+    banner_url: String,
+    auction_house_address: String,
+}
+
+impl<'a> From<models::StoreConfigJson<'a>> for Marketplace {
+    fn from(
+        models::StoreConfigJson {
+            config_address: _,
+            name,
+            description,
+            logo_url,
+            banner_url,
+            subdomain,
+            owner_address: _,
+            auction_house_address,
+            ..
+        }: models::StoreConfigJson,
+    ) -> Self {
+        Self {
+            subdomain: subdomain.into_owned(),
+            name: name.into_owned(),
+            description: description.into_owned(),
+            logo_url: logo_url.into_owned(),
+            banner_url: banner_url.into_owned(),
+            auction_house_address: auction_house_address.into_owned(),
+        }
+    }
+}
+
+#[juniper::graphql_object(Context = AppContext)]
+impl Marketplace {
+    pub fn subdomain(&self) -> String {
+        self.subdomain.clone()
+    }
+
+    pub fn name(&self) -> String {
+        self.name.clone()
+    }
+
+    pub fn description(&self) -> String {
+        self.description.clone()
+    }
+
+    pub fn logo_url(&self) -> String {
+        self.logo_url.clone()
+    }
+
+    pub fn banner_url(&self) -> String {
+        self.banner_url.clone()
+    }
+
+    pub fn auction_house_address(&self) -> String {
+        self.auction_house_address.clone()
+    }
+
+    pub async fn auction_house(&self, context: &AppContext) -> Vec<AuctionHouse> {
+        // let conn = context.db_pool.get().unwrap();
+        // let mut rows: Vec<models::AuctionHouse> = auction_houses::table
+        //     .filter(
+        //         auction_houses::address.eq(any(store_config_jsons::table
+        //             .select(store_config_jsons::auction_house_address)
+        //             .filter(store_config_jsons::subdomain.eq(self.subdomain.clone())))),
+        //     )
+        //     .select(auction_houses::all_columns)
+        //     .limit(1)
+        //     .load(&conn)
+        //     .unwrap();
+
+        // rows.pop().map(Into::into)
```

Comment:
Does this need to still be here?

Reply:
No,this PR needs some changes and is not ready yet